napari/_qt/layers/qt_image_layer.py

```diff
@@ -10,42 +10,46 @@ class QtImageControls(QtLayerControls):
     def __init__(self, layer):
         super().__init__(layer)
 
-        self.layer.events.clim.connect(lambda e: self.clim_slider_update())
+        self.layer.events.contrast_limits.connect(
+            lambda e: self.contrast_limits_slider_update()
+        )
 
-        # Create clim slider
-        self.climSlider = QVRangeSlider(
+        # Create contrast_limits slider
+        self.contrast_limitsSlider = QVRangeSlider(
             slider_range=[0, 1, 0.0001], values=[0, 1], parent=self
         )
-        self.climSlider.setEmitWhileMoving(True)
-        self.climSlider.collapsable = False
-        self.climSlider.setEnabled(True)
+        self.contrast_limitsSlider.setEmitWhileMoving(True)
+        self.contrast_limitsSlider.collapsable = False
+        self.contrast_limitsSlider.setEnabled(True)
```

Comment:
I'd call this `contrastLimitsSlider`. If there's anything worse than camelCase =P it's mixed_underscore_andCamelCase. =P

Reply:
done